### ISSUE TITLE
[epic] Collect simulation data

### ISSUE THREAD

**Issue by mastoffel** [alan-turing-institute/autoemulate#4]
A wide variety of simulations will help to test the package.

- [x]  tsunami simulations 
- [x] cardiovascular simulations 
- [ ] engineering simulations
- [ ] ice sheet simulations
- [ ] infrastructure
- [x] ~~other interesting simulations~~ **edit:** Added to #71

**Comment by mastoffel** [comment]
Few notes on the [cardiovascular simulation data](https://zenodo.org/record/7405335) after talking to Marina:
Easy datasets (fewer parameters, usually LHS sampled):
- atrial_cell_model / ionic
- circadapt
- passive_mechanics
- tiossue_elecrophysiology

Difficult datasets (many parameters, non-linear, non-smooth, non-LHS):
- atrial_cell_model (other than ionic)
- four chamber datasets (very difficult)

**Comment by mastoffel** [comment]
yes, thanks for adding this @kallewesterling and @zackxconti for providing the data!

**Comment by mastoffel** [comment]
Also adding [this repo](https://github.com/paulcon/as-data-sets) for high dimensional datasets.

**Comment by kallewesterling** [comment]
@mastoffel This is where we're waiting for a dataset from BAS? We might want to add that as an issue here to make sure it's on the roadmap.

**Comment by mastoffel** [comment]
> @mastoffel This is where we're waiting for a dataset from BAS? We might want to add that as an issue here to make sure it's on the roadmap.

Yes, still waiting for the BAS dataset.

**Comment by kallewesterling** [comment]
Could you tell me something more about what the "infrastructure" dataset would be above?

As for "other interesting simulations", I think this might be something we can add to #71 as part of how folks can contribute to the package?

**Comment by mastoffel** [comment]
> Could you tell me something more about what the "infrastructure" dataset would be above?
> 

I'm not sure tbh, this should be something from the DT-TRIC infrastructure side, but I haven't found any good datasets here, but will talk to Keith again at some point.

**Comment by kallewesterling** [comment]
What would a "good dataset" mean? I will poke around as well!

**Comment by mastoffel** [comment]
> What would a "good dataset" mean? I will poke around as well!

Simulation input/output for a simulation that's interesting or useful to emulate. Basically a potential use case for `autoemulate`. Optimally, these will have simple scalar inputs and outputs, rather than a mesh or something more complicated.
